specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/Microsoft.EventHub/preview/2023-07-01"
```

Comment:
I don't think you want to specify a version here -- instead {version_status}/{version} I believe (EG does it for a reference) so that you can generate all versions without having to edit this